### PR TITLE
#163519128 Configure the project to use postgres and sequelize

### PR DESCRIPTION
**Pull request by filschristian**
#### What does this PR do?
Configure Authors Haven to use postgres database engine and sequelize as ORM

#### Description of Task to be completed?
- installed sequelize and postgres packages
-Initialized the project to use sequelize
-Established the connection to the database

#### What are the relevant pivotal tracker stories?
#163519128

### CODE REVIEW

**Comment by kimotho-njoki** [comment]
@filschristian Please fix the hound comments

**Comment by filschristian** [comment]
@kimotho-njoki I fixed the vulnerabilities caused by the dependencies. I didn't delete the package. I upgraded it because we saw that we will need it.

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+config/
+database/
```
Comment (kimotho-njoki) [inline .eslintignore]: extra line please at the end of this files

Comment (filschristian) [reply .eslintignore]: @kimotho-njoki the line is now added

**Pull request merged**